lib/triplex.ex

```diff
@@ -197,14 +230,19 @@ defmodule Triplex do
     if reserved_tenant?(tenant) do
       false
     else
-      sql = """
+      sql = case repo.__adapter__ do
+        Ecto.Adapters.MySQL ->
+          "SELECT COUNT(*) FROM #{config().tenant_table} WHERE name = ?"
+        _ ->
```

Comment:
Let's "let it fail" and put `Ecto.Adapters.Postgres` on this case clause ;)

Reply:
Yup